### PR TITLE
[pattgen, dv] Update pattgen checklist.md

### PR DESCRIPTION
**Pull request by tunghoang290780**
Update checklist.md for pattgen v2. This PR should be merge after PR #5127 

Signed-off-by: Tung Hoang <hoang.tung@wdc.com>

### CODE REVIEW

File: hw/ip/pattgen/doc/checklist.md
Comment (weicaiyang) [inline hw/ip/pattgen/doc/checklist.md:176]: Do we meet below requirement for coverage?

SIM_CODE_COVERAGE_V2
Code coverage requirements: line, toggle, fsm (state & transition), branch, assertion: 90%

SIM_FUNCTIONAL_COVERAGE_V2
Functional coverage requirements: coverpoints: 100%, crosses: 75%

**Review by weicaiyang — COMMENTED** [review]
looks like design is still in D1. We need design get to D2 before we can sign off V2

File: hw/ip/pattgen/doc/checklist.md
Comment (tunghoang290780) [reply hw/ip/pattgen/doc/checklist.md:176]: Ops, I think that should be N/A. I have not completed the coverage plan.

**Comment by tunghoang290780** [comment]
> looks like design is still in D1. We need design get to D2 before we can sign off V2

@martin-lueker could you update checklist.md for pattgen D2 if needed

File: hw/ip/pattgen/doc/checklist.md
Comment (weicaiyang) [reply hw/ip/pattgen/doc/checklist.md:176]: then it should be WIP or Not Started

Comment (tunghoang290780) [reply hw/ip/pattgen/doc/checklist.md]: @weicaiyang thanks, fixed it

**Review by weicaiyang — APPROVED** [review]
LGTM, thanks

**Review by rasmus-madsen — APPROVED** [review]
LGTM although I am unsure if this requires an actual review before updating these
@sriyerg  ?

**Comment by tunghoang290780** [comment]
> LGTM although I am unsure if this requires an actual review before updating these
> @sriyerg ?

@rasmus I think it is not necessary for an actual review now. I will ask Martin for D2 first.

**Comment by cindychip** [comment]
Personally I think since this PR does not claim the V2 status for pattgen, should be okay to merge. Once pattgen reaches V2, we can hold a review meeting to go over all the items in this checklist.md. @weicaiyang @sriyerg @udinator any thoughts?

**Comment by weicaiyang** [comment]
> Personally I think since this PR does not claim the V2 status for pattgen, should be okay to merge. Once pattgen reaches V2, we can hold a review meeting to go over all the items in this checklist.md. @weicaiyang @sriyerg @udinator any thoughts?

agree to merge this.. For signing off V2, not sure if it's required to have a review meeting, or just need a PR review. We do have sign-off meeting for V1 and V3. Let me merge this but we can continue the discussion.

**Pull request merged**
